hs_core/models.py

```diff
@@ -86,7 +87,11 @@ def can_add(self, request):
     def can_delete(self, request):
```

Comment:
This looks like it duplicates [hs_core.hydroshare.utils.authorize()](https://github.com/hydroshare/hydroshare/blob/develop/hs_core/views/utils.py#L49).  There should be only one way to assert authorization.